### PR TITLE
Add NSP logging configuration commands - Microsfot.Network

### PR DESCRIPTION
**Pull request by karthik-gumpu**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [x] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by kaushal087** [comment]
Please test these things:

1. Exisitng test cases should not fail
2. New test cases should not fail
3. New varient of commands for exisiting cmdlets should not be generated.
4. New commands along with it's synonym should run successfully.

File: tools/AzPreview/AzPreview.psm1 [hunk 1]
```diff
@@ -0,0 +1,316 @@
+#
```
Comment (VeryEarly) [inline tools/AzPreview/AzPreview.psm1:1]: do not check in unrelated file

File: src/Network/NetworkSecurityPerimeter.Autorest/README.md [hunk 2]
```diff
@@ -31,10 +31,11 @@ require:
 # readme.azure.noprofile.md is the common configuration file
 # - ../../../../tools/SwaggerCI/readme.azure.noprofile.md
   - $(this-folder)/../../readme.azure.noprofile.md
+commit: main
 input-file:
 # You need to specify your swagger files here.
-  - https://github.com/bhupeshbhatt1985/azure-rest-api-specs/tree/1575c2274b36f8f3dd6d3895ac630853b41690f9/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json
-#  - C:\repo\azure-rest-api-specs/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json
+  # - https://github.com/karthik-gumpu/azure-rest-api-specs/blob/31c564921c35aaa371c2f0256c1eb4b73455e2f6/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json
+ - E:\Code\azure-rest-api-specs/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json
```
Comment (VeryEarly) [inline src/Network/NetworkSecurityPerimeter.Autorest/README.md:38]: do not use local spec

**Comment by kaushal087** [comment]
@VeryEarly , we found a bug in this PR. Can you **NOT** release the preview version now? We will raise a new PR by tomorrow for bug fix.
